traitsui/extras/demo.py

```diff
@@ -701,11 +742,18 @@ def get_children_from_datastructure(self):
             cur_path = join(path, name)
             if isdir(cur_path):
                 if self.has_py_files(cur_path):
-                    dirs.append(DemoPath(parent=self, name=name))
+                    dirs.append(
+                        DemoPath(
+                            parent=self,
+                            name=name,
+                            css_filename=join('..', self.css_filename)
+                        )
+                    )
             elif self.use_files:
                 if name != "__init__.py":
                     try:
                         demo_file = self._handle_file(name)
+                        demo_file.css_filename = self.css_filename
```

Comment:
https://github.com/enthought/traitsui/issues/820

Reply:
This should hopefully be resolved now.

Reply:
Thanks!

Don't know if you want to handle this too (sorry for commenting two things on the same line again):

> Would it make sense to move the assignment of css_filename be moved inside _handle_file where demo_file is instantiated? This may help preventing this from being forgotten again.